packages/server-core/src/user/user-inventory/user-inventory.hooks.ts

```diff
@@ -10,7 +12,36 @@ export default {
     get: [],
     create: [],
     update: [disallow()],
-    patch: [],
+    patch: [
+      async (context: HookContext): Promise<HookContext> => {
+        if (context.data.type === 'transfer') {
+          let { privateKey, fromUserAddress, toUserAddress, quantity, walletAmt } = context.data
+          const response = await axios.post(`${config.blockchain.blockchainUrl}/authorizeServer`, {
+            authSecretKey: config.blockchain.blockchainUrlSecret
+          })
+          // KEEP TOKEN
+          const accessToken = response.data.accessToken
+
+          // CALL WALLET API WITH HEADER SETUP
+          const walletData = await axios.post(
+            `${config.blockchain.blockchainUrl}/wallet/send`,
+            {
+              privateKey: privateKey,
```

Comment:
`privateKey` should not be needing to be sent anymore, please remove this. 

also, if fromUserAddress and toUserAddress are no longer being sent from the client request, how does this request have it, and how does the backend know who to sent from and to? 

we also need to check that the user who is sending the request is in fact the user who is logged in.

Reply:
changes done